kornia/geometry/boxes.py

```diff
@@ -564,7 +568,7 @@ def to_mask(self, height: int, width: int) -> torch.Tensor:
             mask = torch.zeros((self._data.shape[0], height, width), dtype=self.dtype, device=self.device)
 
         # Boxes coordinates can be outside the image size after transforms. Clamp values to the image size
-        clipped_boxes_xyxy = cast(torch.Tensor, self.to_tensor("xyxy", as_padded_sequence=True))
+        clipped_boxes_xyxy = cast(Tensor, self.to_tensor("xyxy", as_padded_sequence=True))
```

Comment:
should not need this cast here
```suggestion
        clipped_boxes_xyxy = self.to_tensor("xyxy", as_padded_sequence=True)
        if not isinstance(Tensor, clipped_boxes_xyxy):
             raise
```